build.gradle

```diff
@@ -67,11 +63,6 @@ configurations {
 
 dependencies {
     compile group: 'io.pravega', name: 'pravega-client', version: pravegaVersion
-    compile group: 'io.pravega', name: 'pravega-common', version: pravegaVersion
-    compile group: 'io.pravega', name: 'pravega-shared', version: pravegaVersion
-    compile group: 'io.pravega', name: 'pravega-shared-metrics', version: pravegaVersion
-    compile group: 'io.pravega', name: 'pravega-shared-protocol', version: pravegaVersion
-    compile group: 'io.pravega', name: 'pravega-shared-controller-api', version: pravegaVersion
```

Comment:
I have removed these additional dependencies and ran `./gradlew clean build` and did not see any compile/runtime issues when it runs the test. So I assumed that these dependencies are not required?